bugbug/model.py

```diff
@@ -329,7 +417,14 @@ def classify(
 
                 html = out.getvalue()
 
-            return classes, {"importances": top_importances, "html": html}
+            return (
+                classes,
+                {
+                    "importances": important_features,
+                    "html": html,
+                    "predicted_class": pred_class_index,
```

Comment:
This doesn't seem useful, the caller should be able to get the pred_class_index from `classes`